### PR TITLE
NO-DED Adding one required scope to the Create Agent method

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

As agreed missing scope was added together with its description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/getting-started/app-guides/integrations/index.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ desc: "Streamline your workflow by integrating with supported development servic
 
 # Introduction
 
-This document is a reference of integrations available in the <Placeholder id="DEVELOPER_CONSOLE_URL" />. Connect with different development tools and streamline your workflow by creating repositories directly from the Console and integrating with supported deployment services (coming soon!).
```
Comment (kamilatokarska) [inline src/pages/getting-started/app-guides/integrations/index.mdx:12]: @litwinow please do not pay attention to this one, I just needed to remove it so I took a chance I have this PR😜

File: src/pages/management/configuration-api/index.mdx [hunk 2]
```diff
@@ -202,13 +202,14 @@ Creates a new agent with specified parameters within a license.
...
+|                                  |                                                                                                                                        |
+| -------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| Method URL                       | `https://api.livechatinc.com/v3.5/configuration/action/create_agent`                                                                   |
+| Required scopes                  | `accounts--all:rc` `accounts--all:rw`<sup>**1**</sup> `subscription.lc:rw`<sup>**1**</sup> `accounts.roles.lc--all:rw`<sup>**2**</sup> |
+| [Batch support](#batch-requests) | Yes                                                                                                                                    |
 
-**1)** – to create agents who don't have to wait for approval after creation
+**1)** – to create agents who don't have to wait for approval after creation<br/>
+**2)** – to add a `viceowner` to the call. Required when creating an agent with a role different than `normal` (default)
```
Comment (litwinow) [inline src/pages/management/configuration-api/index.mdx:212]: how about
```
to create an agent with role higher than `normal` (`administrator`, `viceowner` or `owner`)
```
? applies to all versions

Comment (kamilatokarska) [reply src/pages/management/configuration-api/index.mdx:212]: Sounds better, I'll fix it :)

**Comment by ale-kacperczyk** [comment]
We currently don't mention `accounts.roles--all:rw` in the Scopes doc

**Review by litwinow — APPROVED** [review]

**Pull request merged**